app/screens/channel/channel.ios.js

```diff
@@ -25,14 +24,6 @@ const CHANNEL_POST_TEXTBOX_CURSOR_CHANGE = 'onChannelTextBoxCursorChange';
 const CHANNEL_POST_TEXTBOX_VALUE_CHANGE = 'onChannelTextBoxValueChange';
 
 export default class ChannelIOS extends ChannelBase {
-    previewChannel = (passProps, options) => {
-        const screen = 'ChannelPeek';
-
-        peek(screen, passProps, options);
-    };
-
-    optionalProps = {previewChannel: this.previewChannel};
```

Comment:
optional props is no longer used,  so it does not need to be included in line 78. Also there is no need for it here https://github.com/mattermost/mattermost-mobile/blob/master/app/screens/channel/channel_base.js#L271 and here https://github.com/mattermost/mattermost-mobile/blob/master/app/screens/channel/channel_base.js#L321

Reply:
Great catch! Done.